packages/mobile/components/inputs/SearchInput.svelte

```diff
@@ -0,0 +1,24 @@
+<script lang="ts">
+    import { Icon } from 'shared/components'
+    import { localize } from '@core/i18n'
+    import { debounce } from '@core/utils'
+    import { Icon as IconEnum } from '@lib/auxiliary/icon'
+
+    export let value: string
+
+    let searchValue = ''
+
+    $: debounce(() => {
+        value = searchValue
+    })()
+</script>
+
+<input-container class="flex items-center py-2 px-4 rounded-lg bg-gray-50">
+    <Icon icon={IconEnum.Search} classes="mr-2 text-gray-500 dark:text-white" width={21} height={21} />
+    <input
+        bind:value={searchValue}
+        placeholder={localize('general.search')}
+        type="text"
+        class="text-sm bg-transparent text-gray-500 dark:text-white"
+    />
```

Comment:
could we also add an X button to clear the search value? I think this is pretty handy in mobile specially